examples/nlp/language_modeling/conf/bert_pretraining_from_preprocessed_config.yaml

```diff
@@ -0,0 +1,72 @@
+# BERT Pretraining from Preprocessed (tokenized) data
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU, or list with gpu indices
+  num_nodes: 1
+  max_steps: 100 # precedence over max_epochs
+  num_sanity_val_steps: 0 # needed for bert pretraining from preproc
+  replace_sampler_ddp: false # needed for bert pretraining from preproc
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  precision: 16 # 16 to use AMP 
+  amp_level: O1 # O1 or O2 if using AMP 
+  distributed_backend: ddp
+  gradient_clip_val: 0.0
+  row_log_interval: 1
+  val_check_interval: 1.0 # check once per epoch .25 for 4 times per epoch
+  checkpoint_callback: false # provided by exp_manager
+  logger: false # provided by exp_manager
+
+model:
+  language_model:
+    pretrained_model_name: bert-base-uncased # huggingface model name
+    bert_checkpoint: null
+    bert_config:
+      attention_probs_dropout_prob: 0.1
+      hidden_act: gelu
+      hidden_dropout_prob: 0.1
```

Comment:
Is there an easy way for user to train a bert-base with more heads? Somehow that user does not need to copy paste the whole configs here. Is it easy to load from the config file or let HF loads it, then override values from our yaml file instead of just using one of them?
In this way, user can control and update just the ones she likes.

Reply:
Yes, I think it is easy. Right now, if the user specifies a .json huggingface config, then that will be used. If the user wants to instead put the config in the yaml, then that will be used. HF requires a config to passed in order to instantiate the model and we've provided both ways of doing that to the user, from json or from yaml.